crates/sui-indexer/Cargo.toml

```diff
@@ -16,8 +16,14 @@ chrono.workspace = true
 serde_with.workspace = true
 clap.workspace = true
 tap.workspace = true
-diesel.workspace = true
-diesel-derive-enum.workspace = true
+diesel = { features = [
```

Comment:
reading the doc of `i-implement-a-third-party-backend-and-opt-into-breaking-changes`, seems like it's recommended to pin to a minor version?
https://github.com/diesel-rs/diesel/blob/master/guide_drafts/migration_guide.md#removed-most-of-the-non-public-reachable-api

Reply:
also curious what functionalities do we use of this feature?